uptime_service_validation/coordinator/coordinator.py

```diff
@@ -90,6 +89,23 @@ def main():
                 cassandra.close()
 
             # Step 5 checks for forks and writes to the db.
+            state_hash_df = pd.DataFrame(submissions)
+            state_hash_df.columns = [
+                "submitted_at_date",
+                "submitted_at" ,
+                "submitter",
+                "created_at",
+                "block_hash",
+                "remote_addr",
+                "peer_id",
+                "snark_work",
+                "graphql_control_port",
+                "built_with_commit_sha",
+                "state_hash", 
+                "parent",
+                "height",
+                "slot",
+                "validation_error"]
```

Comment:
Since submissions is a `List[Submission]` I think we can replace this with:

```
from dataclasses import asdict

df = pd.DataFrame([asdict(submission) for submission in submissions])
```

however it needs to be tested. :+1: